src/faq/widget.ts

```diff
@@ -10,6 +10,10 @@ import {
 } from 'phosphor/lib/ui/vdom';
 
 import {
+  CommandIDs as AboutCommandIds
```

Comment:
Could you change this to `AboutCommandIDs` since that's what all the other command ID vars look like?

Or do you think we should change all of them from `FooIDs` to `FooIds`?